dbt/adapters/bigquery/connections.py

```diff
@@ -113,6 +116,8 @@ class BigQueryCredentials(Credentials):
         'dataset': 'schema',
         'target_project': 'target_database',
         'target_dataset': 'target_schema',
+        'retries': 'job_retries',
+        'timeout_seconds': 'job_execution_timeout_seconds',
```

Comment:
This is clever! Aliases also affect node-level configs, but I don't see a reason why we can't do it this way